tests/unit/lib/connect-test.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+import connect from 'ember-cli-redux/lib/connect';
+import { module, test } from 'qunit';
+
+module('Unit | Lib | Connect');
+
+function createFakeReduxService(state = {}, dispatch = null) {
+  return Ember.Object.extend({state, dispatch}).create();
+}
+
+test('it works', function(assert) {
+  const subject = connect();
+  assert.ok(subject);
+});
+
+test('statemap strings become one way aliases', function(assert) {
+  const fakeReduxService = createFakeReduxService({ someReduxProp: 'ok' });
+  const stateToComputed = { someProp: 'someReduxProp' };
+  const component = connect(stateToComputed, {}, fakeReduxService)(Ember.Object).create();
+  assert.equal(component.get('someProp'), 'ok');
+});
+
+test('statemaps accept custom functions', function(assert) {
```

Comment:
this would require you to change the test name every time you changed underlying implementation